server/src/test/java/io/crate/planner/operators/LogicalPlannerTest.java

```diff
@@ -569,4 +568,33 @@ public void test_eval_qtf_doesnt_unwrap_non_fetchable_aliases() {
             """
         );
     }
+
+
+    // tracks a bug: https://github.com/crate/crate/issues/13779
+    @Test
+    public void test_prune_outputs_on_group_hash_aggregate() {
+        LogicalPlan plan = sqlExecutor.logicalPlan("""
```

Comment:
FYI, this breaks: https://github.com/crate/crate/pull/13784/files#diff-3423dab235370c0833f0c67b5c08eccf3d3a217fe26c9569571e7c2afaa05e98R70,

Should we investigate here, why 28Mb is not enough?

Reply:
How much larger is it now? Could be due to the used plan which is a bit bigger

Reply:
```
Expecting actual:
  83054520L
to be less than or equal to:
  29360128L 
```